src/provider/CentralServerProvider.tsx

```diff
@@ -380,7 +380,7 @@ export default class CentralServerProvider {
     const tenant = await this.getTenant(tenantSubDomain);
     // Call
     const result = await this.axiosInstance.post(
-      `${this.buildCentralRestServerServiceAuthURL(tenant)}/${ServerAction.PASSWORD_RESET}`,
+      `${this.buildRestServerAuthURL(tenant)}/password/${ServerAction.REST_RESET}`,
```

Comment:
Use REST_PASSWORD_REST with full path and not REST_RESET which means nothing when you read the code!